.vscode/tasks.json

```diff
@@ -18,6 +18,32 @@
             "type": "npm",
             "script": "lint",
             "problemMatcher": "$eslint-stylish"
+        },
+        {
```

Comment:
Confirming you intended this file to be added?

Reply:
This file is not needed so I will delete it.